src/runtime/particle.ts

```diff
@@ -205,6 +205,7 @@ export class Particle {
     throw new Error('A particle needs a description handle to set a decription pattern');
   }
 
+  // TODO(alxr) What are thoughts about marking these methods as `abstract`?
```

Comment:
The Particle class needs to be consumed by non-typescript js.  AFAIK there is existing code that extends Particle that do not implement these in particles/*

Reply:
That's persuasive to me.